doc/publish.yml

```diff
@@ -0,0 +1,60 @@
+site:
+  title: Neo4j Graph Data Science Client
+  url: https://neo4j.com/docs
+  start_page: graph-data-science-client:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../../
```

Comment:
Should this be `url: ../` too?

Reply:
That seems right :) ill update it

Reply:
done

Reply:
The paths should be the same as in preview.yml, yes. This playbook will be needed when we want to publish multiple versions of the client docs